resources/authgear/templates/en/web/authflowv2/settings.html

```diff
@@ -0,0 +1,193 @@
+        {{ $identityArr = $.PreferredUsernames }}{{ $identity = $.PreferredUsername }}
+      {{ end }}
+
+      {{/* Render with given variable */}}
+      {{ template "authflowv2/__settings_item.html"
+        (dict
+          "Label" $label
+          "RedirectURL" $url
+          "Icon" $icon
+          "ContentArr" $identityArr
+        )
+      }}
+
+    {{ end }}
+    {{ end}}
+
+    {{ $url := call $.MakeURL "/settings/identity" }}
+    {{ template "authflowv2/__settings_item.html"
+      (dict
+        "Label" "v2.page.settings.default.button-label-social-accounts"
+        "RedirectURL" $url
+        "Icon" "link"
+        "ContentArr" nil
+      )
+    }}
+    </div>
+  </div>
+
+  {{/* Security */}}
+  <div class="flex flex-col gap-2">
+    <div class="flex flex-col gap-1">
+      <h2 class="settings-title text-start">
+        {{ template "v2.page.settings.default.title-security" . }}
+      </h2>
+      <div class="settings-description text-start">
+        {{ template "v2.page.settings.default.description-security" . }}
+      </div>
+    </div>
+
+    <div class="divide-y">
+      {{ range $.Authenticators }}
+        {{ if and (eq .Type "password") (eq .Kind "primary") }}
+          {{ $url = call $.MakeURL "/settings/change_password" }}
+          {{ template "authflowv2/__settings_item.html"
+            (dict
+              "Label" "v2.page.settings.default.button-label-password"
+              "RedirectURL" $url
+              "Icon" "key"
+              "ContentArr" nil
+            )
+          }}
+        {{ end }}
+      {{ end }}
+
+      {{ if $.ShowPrimaryPasskey }}
+        {{ $url = call $.MakeURL "/settings/passkey" }}
+        {{ template "authflowv2/__settings_item.html"
+          (dict
+            "Label" "v2.page.settings.default.button-label-passkey"
+            "RedirectURL" $url
+            "Icon" "passkey"
+            "ContentArr" nil
+          )
+        }}
+      {{ end }}
+
+      {{ if $.ShowMFA }}
+        {{ $url = call $.MakeURL "/settings/mfa" }}
+        {{ template "authflowv2/__settings_item.html"
+          (dict
+            "Label" "v2.page.settings.default.button-label-mfa"
+            "RedirectURL" $url
+            "Icon" "encrypted"
+            "ContentArr" nil
+          )
+        }}
+      {{ end }}
+
+      {{ if $.ShowBiometric }}
+        {{ $url = call $.MakeURL "/settings/biometric" }}
+        {{ template "authflowv2/__settings_item.html"
+          (dict
+            "Label" "v2.page.settings.default.button-label-biometric-login"
+            "RedirectURL" $url
+            "Icon" "fingerprint"
+            "ContentArr" nil
+          )
+        }}
+      {{ end }}
+
+      {{ $url = call $.MakeURL "/settings/sessions" }}
+      {{ template "authflowv2/__settings_item.html"
+        (dict
+          "Label" "v2.page.settings.default.button-label-signed-in-devices"
+          "RedirectURL" $url
+          "Icon" "verified_user"
+          "ContentArr" nil
```

Comment:
See if we can omit passing `Children`. If yes, then remove all those `"Children" nil`